pkg/util/utils.go

```diff
@@ -111,10 +111,20 @@ func (m MultiError) ToError() error {
 // EscapeStringForSSHUse will parse string and replace
 // shell special characters with escape
 func EscapeStringForSSHUse(s string) string {
-	r := strings.NewReplacer("$", "\\$",
+	r := strings.NewReplacer(
+		"$", "\\$",
 		`"`, `\"`,
 		"`", "\\`",
-		`\`, `\\`)
+		`\`, `\\`,
+		`(`, `\(`,
+		`)`, `\)`,
+		`{`, `\{`,
```

Comment:
We even document using single quotes - https://docs.openshift.org/latest/minishift/using/troubleshooting.html#special-characters-passwords

The things which gets in the way here is the escaping rules of your shell. Once the password is passed on to us, we are dealing with the rules around what and what not is allowed  in terms of passing on the password as part of an ssh command. The rules differ. 

I tried with single quotes and for all I can tell things like ';' or spaces are not a problem.

@budhrg have you actually tested this end to end. I have a feeling you are breaking actual passwords in some cases.